packages/datagateway-common/src/card/cardView.component.tsx

```diff
@@ -713,29 +714,33 @@ const CardView = (props: CardViewProps): React.ReactElement => {
                           information &&
                           information
                             .map((details) => ({
+                              icon: details.icon,
                               // We can say the data key is the label if not defined.
                               label: details.label
                                 ? details.label
                                 : details.dataKey,
-                              content: details.content
-                                ? details.content(entity)
-                                : nestedValue(entity, details.dataKey),
                               // Keep the dataKey in so we can use it for adding the tooltip
                               // once content has been created.
                               dataKey: details.dataKey,
-                              icon: details.icon,
+                              content: details.content
+                                ? details.content(entity)
+                                : nestedValue(entity, details.dataKey),
+                              noTooltip: details.noTooltip ? true : false,
```

Comment:
Probably don't even need the ternary operator here, could just do `noTooltip: details.noTooltip,`

Reply:
Thanks @sam-glendenning, I think I had put that in due to the `noTooltip` being `undefined` when it is not passed as a prop, however, it would work without the ternary operator regardless like you mentioned.